### DISCUSSION TITLE
Do you maintain the whole codebase?

### DISCUSSION THREAD

**Discussion by EwilsonS** [Netflix/conductor#2268] [Q&A] [answered]
I ran across this [conversation](https://github.com/Netflix/conductor/issues/2014#issuecomment-749280291) in issues, where it's said that the MysqlQueueDao is not used or maintained. Is this actually true? I assumed that you all maintain any code that is merged from contributor PRs. I'm asking because my group has created a Sql server module and I notice there is a pr out there doing essentially the same thing. If that pr is merged am I to assume it may not adhere to standards like file structure and dependency management? 

Also, which persistence layers were created and are maintained by the Netflix dev teams?

**Comment by aravindanr** [comment, marked as answer]
Short answer is, we try to.  Long answer is, there is no clear demarcation between community contributions and core supported modules. Before a PR is merged, we ensure that file structure and coding standards are maintained. But we won't know the intricacies of the contribution to enough to fully support it. We typically tag the original authors of the module if there are any support issues in the contributed modules.

> Also, which persistence layers were created and are maintained by the Netflix dev teams?

cassandra-persistence, es*-persistence, redis-persistence are supported by us.

**Comment by james-deee** [comment]
For what it's worth @EwilsonS , we have used Mysql for a few years now and has worked fine for us. I also know others have used Postgres as well. But I do not think the Netflix team maintains those libs.
